forge-gui/res/cardsfolder/upcoming/phyresis_outbreak.txt

```diff
@@ -0,0 +1,9 @@
+Name:Phyresis Outbreak
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Poison | Defined$ Opponent | Num$ 1 | SubAbility$ RepeatPlayers | SpellDescription$ Each opponent gets a poison counter. Then each creature your opponents control get -1/-1 for each poison counter its controller has.
+SVar:RepeatPlayers:DB$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBDebuff
+SVar:DBDebuff:DB$ PumpAll | ValidCards$ Creature.RememberedPlayerCtrl | NumAtt$ -X | NumDef$ -X
+SVar:X:Count$PoisonCounters
```

Comment:
PlayerCountRemembered$PoisonCounters

Needs testing

Reply:
Seems to work fine Thanks 
![image](https://user-images.githubusercontent.com/67333662/213863843-78d7cd4e-d8bc-4187-a27d-ceb5b2945a6c.png)